README.md

```diff
@@ -240,13 +240,10 @@ All properties are mandatory.
 }
 ```
 
-Only the `topic` and properties are mandatory
+Only the `topic` property is mandatory.
 Both `topic` and `payload` can be `Buffer` objects instead of strings.
 `messageId` is mandatory for `qos > 0`.
 
-If `payload` is passed to `generate(packet)` as a string, it will be
-automatically converted into a `Buffer`.
-
```

Comment:
See https://github.com/mqttjs/mqtt-packet/blob/master/writeToStream.js#L258 and https://github.com/mqttjs/mqtt-packet/blob/master/generate.js#L21